docs/local-development-environment.adoc

```diff
@@ -0,0 +1,182 @@
+= Local Development Environment
+
+You can easily test the playbook in a local Virtual Machine that we
+can start and stop quickly, called a _devbox_. This document lays out
+the details on how to setup such devbox.
+
+== Prerequisites
+
+To be able to run the playbook on a devbox several groups of
+requirements needs to be covered.
+
+=== Image preparation requirements
+
+Trento's Ansible playbook requires target systems with specific
+conditions that are not trivial to provide such as:
+
+- The target of the playbook should be SLES4SAP system.
+- The system needs to already be registered with SCC.
+
+To help out with these requirements, there are scripts and auxiliary
+helper files prepared in `devbox` directory. The scripts are
+POSIX-compliant and expect only widely-available tools. The only tool
+to note explicitly is `envsubst` command from `gettext-runtime`
+package. Nevertheless, it should be available in most base-systems.
```

Comment:
About: Nevertheless, it should be available in most base-systems.

Just as note for us it is important that it works on suse, all other distro's are nice to have but not mandatory. As long as it works on suse, no direct need to say it.